Library/Formula/libodb.rb

```diff
@@ -0,0 +1,19 @@
+class Libodb < Formula
+  homepage "http://www.codesynthesis.com/products/odb"
+  url "http://www.codesynthesis.com/download/odb/2.3/libodb-2.3.0.tar.bz2"
+  sha1 "eebc7fa706bc598a80439d1d6a798430fcfde23b"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def caveats; <<-EOS.undent
+        ODB is undel GPL v2 or NCUEL.
+        Please read http://codesynthesis.com/products/odb/license.xhtml before use.
+        EOS
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!